gutenberg.php

```diff
@@ -3,7 +3,7 @@
  * Plugin Name: Gutenberg
  * Plugin URI: https://github.com/WordPress/gutenberg
  * Description: Printing since 1440. This is the development plugin for the block editor, site editor, and other future WordPress core functionality.
- * Requires at least: 6.2
+ * Requires at least: 6.3
```

Comment:
This will also require changing the commit SHA for the performance test. See https://github.com/WordPress/gutenberg/issues/54107#issuecomment-1703397980.

Reply:
The perf tests are already failing on trunk since 

The commit hash was updated [here](https://github.com/WordPress/gutenberg/commit/352ab9993ed4eefd0d11f15dc4146283ec2080ff) but tests are still failing; we probably need to pick another commit 😅

Reply:
I just tried [another commit](https://github.com/WordPress/gutenberg/commit/45ffee3f35cb2209b23b67904b7d065804b6ebe3) based on the logic you used in #54111; let's see if it works 🤞